plugins/destination/duckdb/client/write.go

```diff
@@ -130,13 +130,20 @@ func (c *Client) WriteTableBatch(ctx context.Context, name string, msgs message.
 		return nil
 	}
 	table := msgs[0].GetTable()
+
+	writeStart := time.Now()
 	tmpFile, err := writeTMPFile(table, msgs)
 	if err != nil {
 		return err
 	}
+	c.logger.Debug().Str("table", table.Name).Str("duration", time.Since(writeStart).String()).Msg("write tmp file")
 	defer os.Remove(tmpFile)
 
 	if len(table.PrimaryKeys()) == 0 {
+		copyStart := time.Now()
+		defer func() {
+			c.logger.Debug().Str("table", table.Name).Str("duration", time.Since(copyStart).String()).Msg("copy file to table")
```

Comment:
I've also left these, they are useful in general and don't add overhead compared to how long writing a parquet file takes.